src/App.js

```diff
@@ -1,30 +1,50 @@
-import React from 'react';
+import React, { useState, useEffect } from 'react';
 import './App.css';
-import ReadWriteFirestore from './components/ReadWriteFirestore.js';
+// import ReadWriteFirestore from './components/ReadWriteFirestore.js';
 import {
   BrowserRouter as Router,
   Switch,
   Route,
   NavLink,
+  Redirect,
 } from 'react-router-dom';
+import Home from './pages/Home';
 import List from './pages/List';
 import AddItem from './pages/AddItem';
 
 function App() {
+  const [userToken, setUserToken] = useState(null);
+
+  useEffect(() => {
+    const userTokenJSON = window.localStorage.getItem('userToken');
+
+    if (userTokenJSON) {
+      setUserToken(JSON.parse(userTokenJSON));
+    }
+  }, [userToken]);
+
+  function setToken() {
+    setUserToken(window.localStorage.getItem('userToken'));
+  }
+
   return (
     <div className="App">
-      <ReadWriteFirestore />
       <Router>
-        <div className="linkContainerStyle">
-          <NavLink activeClassName="active" className="navLink" to="/list">
-            List
-          </NavLink>
-          <NavLink activeClassName="active" className="navLink" to="/additem">
-            Add Item
-          </NavLink>
-        </div>
+        {userToken && (
+          <div className="linkContainerStyle">
+            <NavLink activeClassName="active" className="navLink" to="/list">
+              List
+            </NavLink>
+            <NavLink activeClassName="active" className="navLink" to="/additem">
+              Add Item
+            </NavLink>
+          </div>
+        )}
 
         <Switch>
+          <Route exact path="/">
+            {userToken ? <Redirect to="/list" /> : <Home setToken={setToken} />}
+          </Route>
           <Route path="/list">
```

Comment:
**non blocking:** Up to you if you want to implement here or in issue #5! I don't think we capture this as an AC anywhere, but I feel like you should have a redirect on this line if no userToken.

Problem: A user could hit a url with `/list` and be shown a list even though they don't have a user token. See screenshot below:

<img width="1789" alt="Screen Shot 2021-10-19 at 4 34 32 PM" src="https://user-images.githubusercontent.com/3861088/138004920-2321230c-f8a3-4654-a8e6-7d1e5aebf035.png">